Python/Tests/Analysis/AnalysisTest.cs

```diff
@@ -2128,6 +2128,22 @@ public VariableLocation(int startLine, int startCol, VariableType type, string f
             AssertUtil.ContainsExactly(entry.GetTypeIdsByIndex("oar2", text.IndexOf("oar2 =")), BuiltinTypeId.Unicode);
         }
 
+        [TestMethod, Priority(0)]
+        public void StringFormattingV36() {
+            var text = @"
+y = f'abc {42}'
+ry = rf'abc {42}'
+yr = fr'abc {42}'
+fadd = f'abc{42}' + f'{42}'
```

Comment:
```
def f(val):
    print(val)
f'abc {f(42)}'
```

val should be 42 (if you're not going to actually implement this you could leave it commented out)